octoprint_homeassistant/__init__.py

```diff
@@ -121,6 +121,46 @@ def on_after_startup(self):
                     self._on_mqtt_message,
                 )
 
+        # PSUControl helpers
+        psu_helpers = self._plugin_manager.get_helpers(
+            "psucontrol", "turn_psu_on", "turn_psu_off", "get_psu_state"
+        )
+
+        self.psucontrol_enabled = True
+
+        if psu_helpers:
+            self._logger.info("PSUControl helpers found")
+
+            if "get_psu_state" in psu_helpers:
+                self.get_psu_state = psu_helpers["get_psu_state"]
+                self._logger.debug("Setup get_psu_state helper")
+            else:
+                self._logger.error(
+                    "Helper get_psu_state not found, disabling PSUControl integration"
+                )
+                self.psucontrol_enabled = False
+
+            if "turn_psu_on" in psu_helpers:
+                self.turn_psu_on = psu_helpers["turn_psu_on"]
+                self._logger.debug("Setup turn_psu_on helper")
+            else:
+                self._logger.error(
+                    "Helper turn_psu_on not found, disabling PSUControl integration"
```

Comment:
`turn_psu_on` wouldn't be found for most people not using this feature in Octoprint, right?

Reply:
More specifically, is it normal to expect people to hit this condition such that we shouldn't really consider it an error, or are users not using the functionality filtered the the `if psu_helpers:` check? (Looks to be the case, would like to confirm.)